social-insurance-number.js

```diff
@@ -51,6 +52,10 @@
     return this.firstDigit() === TEMPORARY_RESIDENT_FIRST_DIGIT;
   };
 
+  SocialInsuranceNumber.prototype.isBusinessNumber = function() {
+    return this.firstDigit() === BUSINESS_NUMBER_FIRST_DIGIT;
```

Comment:
Is this also true for NEQs?

Reply:
NEQs have their own format, they're 10-digits according to their FAQ: http://www.registreentreprises.gouv.qc.ca/en/faq/

I just want to be able to differentiate between BNs issued by the CRA and SINs to disallow them on the frontend where SINs are expected

Reply:
Whoa!! Dan Godfrey!!!! 😁